frontend/workflows/ec2/src/reboot-instance.tsx

```diff
@@ -89,7 +89,7 @@ const RebootInstance: React.FC<WorkflowProps> = ({ heading, resolverType, notes
   };
 
   return (
-    <Wizard dataLayout={dataLayout} heading={heading}>
```

Comment:
The heading is no longer used elsewhere it seems, so should be safe to remove from the destructuring at the top. I think the case applies for several other files.